buoyant_cloud/README.md

```diff
@@ -0,0 +1,66 @@
+# Agent Check: Buoyant Cloud
+
+## Overview
+
+[Buoyant Cloud][1] provides fully managed Linkerd, right on your cluster.
+
+This integration sends the follow Buoyant Cloud data to Datadog:
+
+- Linkerd health alerts
+- Workload traffic alerts
+- Managed Linkerd events
+- Rollout events
+- Metrics (coming soon)
+
+## Setup
+
+### Installation
+
+You need to have an account at [Buoyant Cloud][1] to use this integration. You
+can also sign up for Buoyant Cloud in the Datadog Marketplace.
+
+### Configuration
+
+1. Log in to your Buoyant Cloud account.
+2. Browse to the [Buoyant Cloud Settings][2] page, and click `Connect Datadog`.
+3. Browse to the [Buoyant Cloud Notifications][3] page.
+4. Click the kebab menu to edit a notificaiton.
+5. In the `Edit notification rule` dialog, in the `Destinations` section, select
+   your Datadog org to have all events matching the notification rule sent to
+   Datadog.
```

Comment:
```suggestion
5. Go to the `Destinations` section and select your Datadog account to send all events matching the notification rule sent to Datadog.
```

Reply:
Are the notification rules already set or is this something the customer needs to do on their end? If so, we should add it as a step